### PR TITLE
Micro-optimize a bunch of stuff for init time

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

I used tracy to find various hot procs and reduce their runtime during init.

This cuts down init by 2-5sec roughly.
- Lazyinits glass table and windoor debris
- Removes unused pda painter init
- Removes useless signal broadcast from vent pumps on init
- Lazy-sorts the verb list
- Replaces IsAdminAdvancedProcCall() with a define in some high-volume procs, since the proc is very simple and could be affected by overhead.

## Why It's Good For The Game

Zoom zoom faster init

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Table glass
image

Verbs work as expected
image
image

Vent pump thing is self explanatory. the signal is not needed or used at init.

</details>

## Changelog
:cl:
code: Optimized init by ~2s with various changes.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -318,16 +322,11 @@
...
-				var/atom/movable/AM = X
-				AM.forceMove(T)
-				debris -= AM
+			release_debris(T)
 	qdel(src)
 
 /obj/structure/table/glass/narsie_act()
 	color = NARSIE_WINDOW_COLOUR
-	for(var/obj/item/shard/S in debris)
-		S.color = NARSIE_WINDOW_COLOUR
```
Comment (itsmeow) [inline code/game/objects/structures/tables_racks.dm:330]: not able to replicate this behavior but it's so niche i don't care

**Comment by PowerfulBacon** [comment]
at least partially atomise out the big ones that might need review comments like verb list and mirage border

**Comment by itsmeow** [comment]
You can read, just review it, its like 5 lines that saves me 10 minutes making PRs

**Comment by PowerfulBacon** [comment]
Bigger PRs get exponentially harder to review since it makes it harder to follow logic around and makes it so that we can't merge any PRs that are ready to go without putting them all together. The review has to be done all in 1 go which means it will take longer for a review to come around since it isn't something I can look over in 2 minutes over the course of the day on mobile.

**Comment by itsmeow** [comment]
it's literally 100 lines

**Review by ike709 — APPROVED** [review]

**Pull request merged**